e2e-tests/admin/tests/dynamic-host-catalog-aws-ent.spec.js

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { test, authenticatedState } from '../../global-setup.js';
+import { expect } from '@playwright/test';
+import { nanoid } from 'nanoid';
+
+import * as boundaryCli from '../../helpers/boundary-cli';
+import { CredentialStoresPage } from '../pages/credential-stores.js';
+import { HostCatalogsPage } from '../pages/host-catalogs.js';
+import { OrgsPage } from '../pages/orgs.js';
+import { ProjectsPage } from '../pages/projects.js';
+import { SessionsPage } from '../pages/sessions.js';
+import { TargetsPage } from '../pages/targets.js';
+
+test.use({ storageState: authenticatedState });
```

Comment:
Any test that might need to logout won't use the state, otherwise, it will affect it for other tests. This includes any of the auth method tests (we test out that the new account that is created works)

Reply:
Does it work if you `test.use` to set the `storageState: null` instead for those tests? It seems like it's more common to want to be authenticated than not in the tests

Reply:
mm. I'll try that out on Monday